src/components/NavBar/index.tsx

```diff
@@ -0,0 +1,118 @@
+'use client';
+
+import { useMemo } from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+import COLORS from '@/styles/colors';
+import { Flex } from '@/styles/containers';
+import { useAuth } from '@/utils/AuthProvider';
+import { useActiveStatus } from '@/utils/hooks';
+import { useProfile } from '@/utils/ProfileProvider';
+import IJPlogo from '~/public/images/ijp-logo.webp';
+import { ProfileButton, SmallLinkButton } from '../Buttons';
+import * as Styles from './style';
+
+type NavLink = {
+  name: string;
+  path: string;
+  active: boolean;
+};
+
+export default function NavBar() {
+  const profile = useProfile();
+  if (!profile) throw new Error('Profile must be defined.');
+
+  const auth = useAuth();
+  if (!auth) throw new Error('Auth must be defined.');
+
+  const authButtonView = useMemo(() => {
+    if (profile.profileReady && auth.userId)
+      return (
+        <ProfileButton href="/settings">
+          {profile.profileData?.first_name || 'Profile'}
+        </ProfileButton>
+      );
+
+    return (
+      <>
+        <SmallLinkButton
+          $primaryColor={COLORS.blueMid}
+          $secondaryColor={COLORS.blueDark}
+          href="/login"
+        >
+          Log In
+        </SmallLinkButton>
+        <SmallLinkButton
+          $primaryColor="white"
+          $secondaryColor={COLORS.blueDark}
+          href="/signup"
```

Comment:
do we want to use CONFIG here? (I believe they were added in the auth styles PR, which hasn't been merged yet, so we may have to add this to CONFIG again in this PR)